src/main/kotlin/com/hedvig/underwriter/web/OnboardingController.kt

```diff
@@ -0,0 +1,50 @@
+package com.hedvig.underwriter.web
+
+import com.hedvig.underwriter.model.OnboardingRepository
+import com.neovisionaries.i18n.CountryCode
+import java.util.UUID
+import org.springframework.http.HttpStatus
+import org.springframework.web.bind.annotation.GetMapping
+import org.springframework.web.bind.annotation.PathVariable
+import org.springframework.web.bind.annotation.PostMapping
+import org.springframework.web.bind.annotation.RequestBody
+import org.springframework.web.bind.annotation.RequestMapping
+import org.springframework.web.bind.annotation.RestController
+import org.springframework.web.server.ResponseStatusException
+
+@RequestMapping("/onboarding")
```

Comment:
Not underscored because I intend for this type to be publicly accessible through giraffe.

Reply:
Maybe it should be `/underwriter/onboarding` if we ever want to move towards uniform path prefixes for a service?

Reply:
What happened to external? 🤔 I honestly don't know how one should do this
```suggestion
@RequestMapping("/external/onboarding")
```

Reply:
I don't recall seeing that pattern anywhere? From what I can see in the prod-env configmap, there is no discernible pattern at all between public endpoints. The only thing they have in common is to not have the underscore prefix.

I am previously used to letting each service have a single path prefix that denotes its domain, and only have a single redirect entry for that entire path. So like `path: /underwriter/** -> url: underwriter/`

Reply:
I prefer it the way it is FWIW

Reply:
I think it was you who wrote `external` @bystam in `product-pricing`?

Reply:
I tried searching for `external` in the request mappings of most our big services, and couldn't find anything 🤔 ? I also don't see that word named anywhere in `prod-env/prod/prod-cluster/default/api-gateway/configmap.yml` for instance.